src/java.desktop/share/classes/java/awt/font/NumericShaper.java

```diff
@@ -112,25 +112,22 @@
  * <caption>NumericShaper constants precedence</caption>
  * <thead>
  *   <tr>
- *     <th scope="col">Unicode Range
- *     <th scope="col">{@code NumericShaper} Constants
- *     <th scope="col">Precedence
+ *     <th scope="col" id="colheader1">Unicode Range
+ *     <th scope="col" id="colheader2">{@code NumericShaper} Constants
+ *     <th scope="col" id="colheader3">Precedence
```

Comment:
Shall we use more descriptive ids?
```suggestion
 *     <th scope="col" id="nsUnicodeRangeHeader">Unicode Range
 *     <th scope="col" id="nsConstantsHeader">{@code NumericShaper} Constants
 *     <th scope="col" id="nsPrecedence">Precedence
```

Reply:
Not relevant anymore given last change

Reply:
Why is it not relevant? You're still using the ids?
I see you've removed the `header` attribute but I still `id` attributes. Will you be removing the id attributes too?